doc/cookbook/source/examples/distance/cosine.rst

```diff
@@ -0,0 +1,40 @@
+==================
+Cosine	 Distance
+==================
+
+The Cosine distance for real valued features is the similarity between two vectors by finding their angle.
+
+.. math::
+
+    d({\bf x},{\bf x'})= 1-\frac{\sum_{i=1}^{n}{\bf x_i}.{\bf x'_i}}{\sqrt{\sum_{i=1}^{n}{\bf {x_i}^2}\sum_{i=1}^{n}{\bf {x'_i}^2}}}
```

Comment:
u mean d instead on n for dimension of vector. But that would match with 'd' for distance measure.